www/search

```diff
@@ -1088,6 +1088,10 @@ if ($api_mode) {
                 $item['coverArtLink'] =
                     get_root_url() . "coverart?id=$id&version={$row['pagevsn']}";
             }
+            // Include estimated play time if available
```

Comment:
The code is self-explanatory so I don't think this comment is adding anything

Reply:
Deleted